### PR TITLE
Bug 1828691: Add support for running tests in a disconnected environment.

### PR DESCRIPTION
**Pull request by fedepaol**

### CODE REVIEW

**Comment by fedepaol** [comment]
/hold
Let's wait for images on openshift-kni

File: test/util/images/images.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+var registry string
+var images map[string]imageLocation
+
+func init() {
+	registry = os.Getenv("IMAGE_REGISTRY")
+
+	images = map[string]imageLocation{
+		"testutils": {
+			name:    "cnftest-utils",
+			registy: "quay.io/fpaoline/",
```
Comment (mmirecki) [inline test/util/images/images.go:18]: Do we have a dockerfile for this anywhere?

Comment (fedepaol) [reply test/util/images/images.go:18]: Yes, all the dockerfiles are [here](https://github.com/openshift-kni/cnf-features-deploy/tree/master/cnf-tests/testimages) and we are in the process of mirroring those images under the openshift-kni quay repo, which is the reason of the hold

File: test/conformance/tests/vfs_assignement.go [hunk 2]
```diff
@@ -25,6 +25,8 @@ import (
 var _ = Describe("[sriov] pod", func() {
 	var sriovInfos *cluster.EnabledNodes
 	execute.BeforeAll(func() {
+		Expect(clients).NotTo(BeNil(), "Client misconfigured, check the $KUBECONFIG env variable")
+
```
Comment (SchSeba) [inline test/conformance/tests/vfs_assignement.go:29]: nit: remove this empty line please to be consistent

**Pull request merged**